docs/content/reference/provider-infra/aws.md

```diff
@@ -0,0 +1,63 @@
+
+## AWS Components and Hypershift
+
+As you've seen, we've been showing all the components created, managed, not managed, etc... in the Hypershift life cycle. In this section we want to dissect who creates what and what not. It contains 4 stages:
+
+- Pre-required and Unmanaged Hosted Cluster AWS account
```

Comment:
Let's add one more section
Infra pre-required and unmanaged for hypershift operator in an arbitrary AWS account:
- s3 bucket
- route53 zone